### ISSUE TITLE
Include only whitelist domains

### ISSUE THREAD

**Issue by AmirAref** [nadoo/glider#422]
Hi,
I’m not sure if the feature I’m looking for already exists in Glider or not. If it does, please help me understand how to use it. If it doesn’t exist yet, I think it would be a good feature to add.

I would like to have a rule file where the listeners only include requests from specific domains, while allowing all other requests to connect directly. This is essentially the opposite of the `bypass` rule.

For example:
```
include=whitelist.conf
```
whitelist.conf:
```
domain=some.example.com
```
This means that only requests to `some.example.com` will go through the proxy.

Thank you for your effort!

**Comment by AmirAref** [comment]
.

**Comment by sergeevabc** [comment]
Yes, how to do I route all *.com domains via forwarder, except for `seanconnery.com` ?

Right now I can create a rule file with the following line: `domain=com`.
But how to exclude/whitelist some com domain?
